Lint --fix ignored files

pwltr opened the pull request:
Linted some files that where ignored by xo

AaronDewes reviewed (approved):
I don't know why this was even in there, probably a leftover from Umbrel or Casa.

(Edit: It was me, we had a top-level LND dir and I only wanted to ignore that, I didn't know I had to put a / when I made that commit)

pwltr commented:
> I don't know why this was even in there, probably a leftover from Umbrel or Casa.
> (Edit: It was me, we had a top-level LND dir and I only wanted to ignore that, I didn't know I had to put a / when I made that commit)

I assumed it was something like that, I looked for other lnd folders but there was only this one so I removed the .ignore entry.

Full disclosure for this commit, it was mostly normal lint errors but also 2 instances of "no-confusing-void-expression" which might have also been bugs but I don't think the responses are used anywhere.

closeChannel return Promise<void>, so this is basically assignment to void:
```ts
ctx.body = await lightningLogic.closeChannel(fundingTxId, index, force);
```

changed to this:
```ts
await lightningLogic.closeChannel(fundingTxId, index, force);
ctx.body = {success: true};
```

and a second case like this.
